### PR TITLE
Overriding source.st excluding the data-md-source attribute.

### PR DESCRIPTION
**Pull request by samueleresca**
Removing data-md-source to avoid external 3rd party calls to GH.

See: #110

### CODE REVIEW

File: theme/src/main/assets/partials/source.st [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<a href="$page.properties.("material.repo")$"
+   title="Go to repository"
+   class="md-source">
+  <div class="md-source__icon">
+    <i class="fa fa-$page.properties.("material.repo.type")$"></i>
+  </div>
+  <div class="md-source__repository">
+    $page.properties.("material.repo.name"); null="'material.repo.name' not defined"$
+  </div>
+</a>
```
Comment (pjfanning) [inline theme/src/main/assets/partials/source.st:35]: 2 minor issues:

* could you add a line end at the end of the file?
* could you update the LICENSE file to mention this new file? - it mentions other files that have the same origin as this

Comment (samueleresca) [reply theme/src/main/assets/partials/source.st]: Updated

**Comment by pjfanning** [comment]
I tried this out on my laptop and it seems to work well. The GitHub link appears but we don't seem to get the api.github.com calls.

**Review by Roiocam — COMMENTED** [review]
The solution looks good.

**Pull request merged**